site/content/entry/yuzu-progress-report-dec-2021/index.md

```diff
@@ -0,0 +1,371 @@
+before, considerably reducing the boot times of the emulator itself.
+
+As previously said, german77 continues to work in making `Ring Fit Adventure` playable. 
+He has [stubbed the](https://github.com/yuzu-emu/yuzu/pull/7524) `SetNpadCaptureButtonAssignment`, `ClearNpadCaptureButtonAssignment`, `ListAlarmSettings`, and 
+`Initialize` services, and [added support](https://github.com/yuzu-emu/yuzu/pull/7525) for the `notif:a` service.
+
+With all his changes, the current Early Access build when writting this article can boot and play the first stage of the game:
+
+{{< imgs
+	"./ringfit.mp4| Ring Fit Adventure"
+  >}}
+
+[Tatsh](https://github.com/Tatsh) [added NSP and XCI file association to Linux](https://github.com/yuzu-emu/yuzu/pull/7609).
+Thanks!
+
+[Tachi107](https://github.com/Tachi107) [updated cubeb](https://github.com/yuzu-emu/yuzu/pull/7527) and removed now deprecated functions.
+Cleaner is always better, thanks!
+
+[heinermann](https://github.com/heinermann) fixed a crash that would occur when the 
+[emulation was paused and the window was out of focus](https://github.com/yuzu-emu/yuzu/pull/7506). 
+Thank you!
+
+[jbeich](https://github.com/jbeich) changed the building configuration so that 
+[VA-API, one of the video decoding APIs of Linux, is enabled on Unix systems](https://github.com/yuzu-emu/yuzu/pull/7602), allowing the users who want to build targeting 
+BSD or other Unix-based systems to use hardware acceleration for video decoding.
+
+This is just one of several PRs jbeich wrote to help yuzu work on BSD systems, thank you for your contributions!
+
+## UI changes
+
+The favourites row in yuzu’s game list was always expanded, even if the user collapsed it.
+epicboy [added a persistent setting](https://github.com/yuzu-emu/yuzu/pull/7570) to remember the user preference between launches.
+
+One of the most common issues users face is lack of Vulkan support on their PC. Not lack of hardware support, but instead missing software support caused by outdated GPU 
+drivers or poorly coded/outdated Vulkan injections.
+
+Our old error popup didn’t reflect this so [your writer](https://github.com/goldenx86), with his total lack of coding skills, 
+[decided to change it](https://github.com/yuzu-emu/yuzu/pull/7532).
+
+{{< single-title-imgs
+    " "
+    "./vidbug.png"
+    "./vidfix.png"
+    >}}
+
+This is a complex issue and the main reason Vulkan is not the default API in use.
+Users of old laptops with AMD and Intel integrated GPUs tend to use the driver shipped by either the laptop vendor or Windows Update. 
+In both cases those drivers are most likely years old (yuzu can run on AMD GPUs from 2012) and either lack Vulkan support at all, or only support a portion of what’s 
+needed to run yuzu.
+Also, since laptops by default connect the display directly to the integrated GPU, that’s the first Vulkan driver that will be seen, so it’s critical to have the latest 
```

Comment:
```suggestion
Also, since laptops, by default, connect the display directly to the integrated GPU, that’s the first Vulkan driver that will be seen, so it’s critical to have the latest 
```